apps/judicial-system/web/pages/krafa/domkrofur-og-lagaakvaedi/[id].tsx

```diff
@@ -1,3 +1,3 @@
-import { StepThree } from '@island.is/judicial-system-web/src/routes/Prosecutor/CreateDetentionRequest/StepThree/StepThree'
+import { StepThree } from '@island.is/judicial-system-web/src/routes/Prosecutor/CustodyRequest/StepThree/StepThree'
```

Comment:
This path is also used the travel bans, so perhaps we should change to a name that reflects that. Don't know what that should be though.